tests/test_segment.py

```diff
@@ -215,6 +256,25 @@ def process_func(signal, sampling_rate):
     result = segment.process_signal_from_index(signal, sampling_rate, index)
     pd.testing.assert_index_equal(result, expected)
 
+    # segmented index with relative paths: series and dataframe
+    table = audformat.Table(index)
+    table["values"] = audformat.Column()
+    table.set({"values": [0, 1, 2]})
+    expected_series = pd.Series(table.get()["values"].values,
+                                index=expected,
+                                name="values",
+                                dtype=np.int64)
```

Comment:
This seems not to test if the labels are correctly assigned if the number of segments increases.
I guess it would be nice to add an additional test that explictly checks only for this, e.g. by using the process function, dataframe, and expected dataframe as input.